sqlmesh/core/snapshot/evaluator.py

```diff
@@ -797,6 +799,19 @@ def _create_snapshot(
             else:
                 dry_run = len(deployability_flags) == 1
                 for is_table_deployable in deployability_flags:
+                    if (
+                        is_table_deployable
+                        and snapshot.model.forward_only
+                        and not is_snapshot_representative
```

Comment:
how can a table be deployable but not represenatitve?

Reply:
It's a deployable table for a non-deployable snapshot. That can be the case